java/src/test/java/io/cucumber/gherkin/GherkinDocumentBuilderTest.java

```diff
@@ -52,7 +52,16 @@ void parses_rules() {
                 "    The rule B description\n" +
                 "\n" +
                 "    Example: Example B\n" +
-                "      Given b";
+                "      Given b\n" +
+                "\n" +
+                "  @someTag\n" +
+                "  Scenario: with examples\n" +
+                "      Given the <value> minimalism\n" +
+                "\n" +
+                "      Examples:\n" +
+                "      | value |\n" +
+                "      | 1     |\n" +
+                "      | 2     |";
```

Comment:
I don't understand what motivated this change. This test now looks unfocused. It now tests rules in combination with scenario outlines and tags and backgrounds. But only to a limited degree.

And I would expect these to (already) be part of the acceptance test suite.

Reply:
I don't remember the rationale behind this addition, so I reverted to the original test case.